vignettes/howto-dm-db.Rmd

```diff
@@ -271,7 +271,7 @@ DBI::dbDisconnect(destination_db)
 DBI::dbDisconnect(my_db)
 ``````
 
-## Conclusion
+## Conclusion {#conclusion}
```

Comment:
why is the ID needed, I suspect it's not for pkgdown?

Reply:
No, not for pkgdown. 

I am including this only for consistency because the Conclusion sections of the rest of the vignettes and also other sections in the same vignette have an ID.

Reply:
Thanks, could you please open an issue about this? I wonder why there are IDs.

Reply:
Done. 

But, whichever way we decide, I think this can be handled in a separate PR.